internal/oci/container.go

```diff
@@ -755,3 +756,18 @@ func (c *Container) SetResources(s *specs.Spec) {
 func (c *Container) GetResources() *types.ContainerResources {
 	return c.resources
 }
+
+// SetPlatformForImage sets the platform of the image for container.
+func (c *Container) SetPlatformForImage(os, arch string) { //nolint:gocritic
+	c.platform = fmt.Sprintf("%s/%s", os, arch)
+}
+
+// GetRuntimePath returns the runtime path for container.
+func (c *Container) GetRuntimePath(r *runtimeOCI) string {
```

Comment:
nit: usually go recommends against prepending getters with "Get" instead just going for the variable it's getting: `RuntimePath()`

That said, it may be more self documenting if we mention `RuntimePathGivenPlatform` or something